src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/gerritnotifier/ParameterExpander.java

```diff
@@ -127,12 +129,77 @@ public String getBuildStartedCommand(Run r, TaskListener taskListener,
             }
         }
 
+        parameters.put("BUILDURL", jenkins.getRootUrl() + r.getUrl());
         parameters.put("STARTED_STATS", startedStats.toString());
 
         return expandParameters(gerritCmd, r, taskListener, parameters);
     }
 
     /**
+     * Gets the expanded string to send to Gerrit for a build-started event.
+     * @param runs the builds.
+     * @param taskListener the taskListener.
+     * @param event the event.
+     * @param stats the statistics.
+     * @return the "expanded" command string.
+     */
+    public String getBuildsStartedCommand(List<Run> runs, TaskListener taskListener,
+                                         ChangeBasedEvent event, BuildsStartedStats stats) {
+
+        String gerritCmd = config.getGerritCmdBuildStarted();
+
+        Integer minCodeReview = Integer.MAX_VALUE;
+        Integer minVerified = Integer.MAX_VALUE;
+
+        StringBuilder aggregatedBuilds = new StringBuilder();
+
+        int offset = runs.size() - 1;
+
+        for (Run r : runs) {
+            GerritTrigger trigger = GerritTrigger.getTrigger(r.getParent());
+
+            minCodeReview = Math.min(minCodeReview, getBuildStartedCodeReviewValue(r));
+            minVerified = Math.min(minVerified, getBuildStartedVerifiedValue(r));
+
+            aggregatedBuilds.append("\n\n");
+
+            String buildStartMessage = trigger.getBuildStartMessage();
+            if (buildStartMessage != null && !buildStartMessage.isEmpty()) {
+                aggregatedBuilds.append("\n\n").append(buildStartMessage);
+            }
+
+            String buildUrl = jenkins.getRootUrl() + r.getUrl();
+            aggregatedBuilds.append(buildUrl);
+
+            if (stats.getTotalBuildsToStart() > 1) {
+                aggregatedBuilds.append(" ");
+                aggregatedBuilds.append(stats.stringWithOffset(offset));
+                aggregatedBuilds.append(" ");
+                offset--;
+            }
+
+            if (config.isEnablePluginMessages()) {
+                for (GerritMessageProvider messageProvider : emptyIfNull(GerritMessageProvider.all())) {
+                    String extensionMessage = messageProvider.getBuildStartedMessage(r);
+                    if (extensionMessage != null) {
+                        aggregatedBuilds.append("\n\n").append(extensionMessage);
+                    }
+                }
+            }
+        }
+
+        Map<String, String> parameters = createStandardParameters(event,
+                minCodeReview,
+                minVerified,
+                Notify.ALL.name());
+
+        parameters.put("BUILDURL", "");
```

Comment:
I did it, because it was already like that implemented for `Completed` message (it just ignored `BUILDURL`) completely :). I don't want to touch template part in my PR, but I can add more details in `help` section.